integration_test/run_adapter_and_wait.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I think this file should be deleted, it's a duplicate

Reply:
Right, deleted it.